dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -263,10 +265,10 @@ static Configuration configureFreemarker() {
    * Returns mapping from the Maven coordinates of BOM members to jar files that are in the
    * dependency tree of the BOM members.
    */
-  private static ImmutableSetMultimap<String, Path> bomMemberToJars(
+  private static ImmutableSetMultimap<String, ClassPathEntry> bomMemberToJars(
       ClassPathResult classPathResult) {
```

Comment:
For later: we could perhaps push a lot of this into ClassPathResult whihc could become simply Classpath. Hide a lot of the implementation details.